src/screens/Gurbani/BottomBar.tsx

```diff
@@ -47,7 +49,11 @@ const BottomBar = () => {
 
   return (
     <SafeAreaView style={styles.root} edges={[ 'bottom', 'left', 'right' ]}>
-      <View style={styles.background} />
+
+      <LinearGradient
+        style={styles.background}
+        colors={[ 'rgba(26,26,26,0)', 'rgba(26,26,26,.9)', 'rgba(26,26,26,1)' ]}
```

Comment:
[performance] given this is a constant, let's extract this from the component - that way the array reference does not change on each render as it won't be recreated

Reply:
yeah, all right! I was thinking about that while I was writing this down! Going to move it out!